### PR TITLE
fix float learning rate bug

### PR DESCRIPTION
**Pull request by vslyu**
learning rate(supported) : float or string write such as 1e-3 ...

### CODE REVIEW

File: core/model.py [hunk 1]
```diff
@@ -177,6 +177,12 @@ def optimizer(self):
         opt_name = envs.get_global_env("hyper_parameters.optimizer.class")
         opt_lr = envs.get_global_env(
             "hyper_parameters.optimizer.learning_rate")
+        if not isinstance(opt_lr, (int, float, Variable)):
+            try:
+                opt_lr = float(opt_lr)
+            except ValueError:
+                raise ValueError("learning_rate%s must be in int or float type"
```
Comment (seiriosPlus) [inline core/model.py:184]: 格式是否有问题？

File: core/model.py [hunk 2]
```diff
@@ -177,6 +177,12 @@ def optimizer(self):
         opt_name = envs.get_global_env("hyper_parameters.optimizer.class")
         opt_lr = envs.get_global_env(
             "hyper_parameters.optimizer.learning_rate")
+        if not isinstance(opt_lr, (int, float, Variable)):
```
Comment (seiriosPlus) [inline core/model.py:180]: why `if not isinstance` ?

File: core/model.py (hunk 1 above)
Comment (vslyu) [reply core/model.py:184]: exit error message as follow:
image

**Comment by seiriosPlus** [comment]
merged #197
